SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -146,27 +145,73 @@ void USpatialSender::GainAuthorityThenAddComponent(USpatialActorChannel* Channel
 			// adding the subobject.
 			PendingSubobjectAttachment->PendingAuthorityDelegations.Add(ComponentId);
 			Receiver->PendingEntitySubobjectDelegations.Add(
-				MakeTuple(static_cast<Worker_EntityId_Key>(Channel->GetEntityId()), ComponentId),
+				MakeTuple(static_cast<Worker_EntityId_Key>(EntityId), ComponentId),
 				PendingSubobjectAttachment);
 
-			EntityACL->ComponentWriteAcl.Add(Info->SchemaComponents[Type], AuthoritativeWorkerRequirementSet);
+			NewComponentIds.Add(ComponentId);
 		}
 	});
 
-	FWorkerComponentUpdate Update = EntityACL->CreateEntityAclUpdate();
-	Connection->SendComponentUpdate(Channel->GetEntityId(), &Update);
+	// If the load balancer is enabled and we're not EntityACL authoritative, we need to use the ComponentPresence
```

Comment:
If we have authority over the ACL we still need to keep this the presence component up to date.

Reply:
Can we not just update the presence component regardless and have one path? Do we care about the single server saving on this enough to have two large branches in a function like this?

Reply:
Agreed presence should be updated regardless of ACL auth